src/Spryker/Client/RabbitMq/Model/Connection/Connection.php

```diff
@@ -41,17 +41,22 @@ class Connection implements ConnectionInterface
      * @param \PhpAmqpLib\Connection\AMQPStreamConnection $streamConnection
      * @param \Spryker\Client\RabbitMq\Model\Helper\QueueEstablishmentHelperInterface $queueEstablishmentHelper
      * @param \Generated\Shared\Transfer\QueueConnectionTransfer $queueConnection
+     * @param bool $runtimeSettingUpDisabled
      */
     public function __construct(
         AMQPStreamConnection $streamConnection,
         QueueEstablishmentHelperInterface $queueEstablishmentHelper,
-        QueueConnectionTransfer $queueConnection
+        QueueConnectionTransfer $queueConnection,
+        bool $runtimeSettingUpDisabled = true
```

Comment:
Why don't we put $runtimeSettingUpDisabled to the $queueConnection, especially because of in the end `self::setupQueuesAndExchanges()` uses $this->queueConnectionConfig property?
IMO it would be more consistent, would't it?

Reply:
It is not represented in QueueConnectionTransfer because this config is not only for connection for a particular store but for all connections. Or if you mean why I don't provide `\Spryker\Client\RabbitMq\RabbitMqConfig` it is because such change breaks BC.

Reply:
Chaged